internal/xds/bootstrap/bootstrap.go

```diff
@@ -28,6 +28,10 @@ const (
 
 	// DefaultXdsServerPort is the default listening port of the xds-server.
 	DefaultXdsServerPort = 18000
+
+	envoyReadinessAddress = "0.0.0.0"
```

Comment:
@arkodg - I'm not sure it matters for this PR but does EG at this point claim to work in an IPV6 only environment?

If so then I don't believe `0.0.0.0` would work and I think its something like `::` for IPV6 (_I'm definitely not an IPv6 expert so could be wrong_).

Maybe we just need to capture it as a separate issue but thought I would call it out just in case.

Reply:
FYI: https://github.com/envoyproxy/gateway/issues/184

Reply:
@zirain - cool thanks for pointing out! It looks my comment is already captured so no worries from me.